sycl/include/CL/sycl/property_list.hpp

```diff
@@ -172,6 +178,10 @@ class context_bound
 namespace queue {
 class enable_profiling
     : public detail::Prop<detail::PropKind::QueueEnableProfiling> {};
+
+class in_order : public detail::Prop<detail::PropKind::InOrder> {};
+
+class out_of_order : public detail::Prop<detail::PropKind::OutOfOrder> {};
```

Comment:
Isn't `in_order` property is enough?
SYCL queue is out of order by default.

NOTE: `OutOfOrder` is not tested.

Reply:
Yes, but it seemed like having symmetry was nice.

Reply:
It might be worth adding a document with an extension proposal and finalizing it first with @Pennycook, who asked for re-design https://github.com/intel/llvm/issues/801.